server/controllers/login.js

```diff
@@ -13,22 +13,22 @@ exports.login = (req, res) => {
   })
     .then((result) => {
       if (!result) {
-        return res.status(401).json({ error: 'user not found' });
+        return res.status(401).json({ success: 'false', error: 'user not found' });
```

Comment:
Why you added success message inside each response! you already handled the errors ```error: 'msg' ```

Reply:
done